docs/writing-tests/integrations/vitest-addon.mdx

```diff
@@ -62,7 +60,7 @@ For some project setups, the `add` command may be unable to automate the addon a
 
 1. Make sure Vite and Vitest are configured in your project.
 1. Configure Vitest to use [browser mode](https://vitest.dev/guide/browser/).
-1. Install the addon, `@storybook/addon-vitest`, in your project and [register it in your Storybook configuration](http://storybook.js.org/docs/addons/install-addons#manual-installation).
+1. Install the addon, `@storybook/addon-vitest`, in your project and [register it in your Storybook configuration](../addons/install-addons.mdx#manual-installation).
```

Comment:
Another broken link here (this is deeply nested, should be another one level up). When you can re-check this page again for these issues.